### ISSUE TITLE
IED planting.

### ISSUE THREAD

**Issue by MetalHead89250** [acemod/ACE3#7015]
**Arma 3 Version:** `1.92` (stable)
**CBA Version:** `3.11.2` (stable)
**ACE3 Version:** `3.12.6` (stable)

**Mods:**
```
- CBA_A3
- ace
- acex
- blastcore
- tpw
- JSRS
- ASR
```

**Description:**
- There's a problem with planting small IED dug-in variant. If a player plants it and selects pressure plate as a trigger, the IED changes itself into large dug-in variant. I've managed to localize source of the problem - there was a mistake in class definition of that specific explosive
in config.bin file of the ace_explosives module. Fixing that solved the problem - small IED is now correctly spawned after selecting pressure plate trigger.

**Steps to reproduce:**
- Place any unit in editor, add small IED (dug-in), start mission, place it anywhere, select pressure plate trigger.

**Where did the issue occur?**
- Basically anywhere.

**Additional information:**

**RPT log file:**
http://www.mediafire.com/file/49myhxrb73qyzyf/ace_explosives.pbo/file
It's a corrected ace_explosives.pbo, not a logfile, but I think that'd be more useful :)

**Comment by TheMagnetar** [comment]
You can also make a pull request
